README.md

```diff
@@ -297,18 +296,35 @@ times for multiple relays. Use whichever method suits your preferences.
 
 These two MEV-Boost commands are equivalent:
 
-```
-./mev-boost -mainnet -relay-check \
+```bash
```

Comment:
Same for this one and the others.

Reply:
https://github.com/flashbots/mev-boost/pull/420